cron/closeIndices.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+# set this to run as a cronjob (after backup has completed)
+# to regularly remove indexes
+
+# .conf file will determine what indexes are operated on
+# Create a starter .conf file with backupDiscover.py
+
+import sys
+from datetime import datetime, timedelta
+from configlib import getConfig, OptionParser
+
+from mozdef_util.utilities.logger import logger
+from mozdef_util.utilities.toUTC import toUTC
+from mozdef_util.elasticsearch_client import ElasticsearchClient
+
+
+def esCloseIndices():
+    logger.debug('started')
+    try:
+        es = ElasticsearchClient((list('{0}'.format(s) for s in options.esservers)))
+        indices = es.get_indices()
+    except Exception as e:
+        logger.error("Unhandled exception while connecting to ES, terminating: %r" % (e))
+
+    # examine each index pulled from get_indice
+    # to determine if it meets aging criteria
+    for index in indices:
+        if 'events' in index:
+            index_date = index.rsplit('-', 1)[1]
+            logger.debug("Checking to see if Index: %s can be closed." % (index))
+            month_ago_date = toUTC(datetime.now()) - timedelta(days=options.index_age)
```

Comment:
You need to convert options.index_age to an integer, or else this line stack traces.

Reply:
We can also move this line to right before the for loop (line 32), since we only have to set this variable once.

Reply:
Good call, though in my tests it behaves as an integer, so not sure what changes between my environment and yours.